### DISCUSSION TITLE
fill missing error and attribute error, pls help me to short it out.

### DISCUSSION THREAD

**Discussion by Biswajit8280** [anujdutt9/Disease-Prediction-from-Symptoms#15] [General]
hi @anujdutt9 , i am curently work on that disease prediction web project. i am cloned your project into vs code ,when i run main.py and infer.py files then an missing file error (FileNotFoundError: [Errno 2] No such file or directory: './saved_model/random_forest.joblib')  and  attribute error (AttributeError: 'DiseasePrediction' object has no attribute 'config')  will show.

pls help me to short it out.
i will provide you screen short of both error.
![Screenshot 2024-08-14 172904](https://github.com/user-attachments/assets/6984e174-1442-4c7a-8046-53ed3645c140)
![Screenshot 2024-08-14 173019](https://github.com/user-attachments/assets/52f771f5-33d1-4d47-b4ad-d96c6f4c0f17)

**Comment by anujdutt9** [comment]
Hi @Biswajit8280 `random_forest.joblib` is the trained model. You can run the respective classifier on the dataset and it'll generate this file. Then use this file for inference.
